.github/CONTRIBUTING.md

```diff
@@ -8,4 +8,15 @@ Welcome to contributing in **Open Source Programs**!
 - When submitting a PR, make your PR as descriptive as possible, follow the template, add screenshots and relevant information.
 - When you are finished with your work, please squash your commits otherwise we will squash them on your PR. This will help maintain a clean commit history.
 - For in-code documentation, follow [this guide](https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings).
-- The full guidelines for python can be found [here](https://github.com/google/styleguide/blob/gh-pages/pyguide.md).
\ No newline at end of file
+- The full guidelines for python can be found [here](https://github.com/google/styleguide/blob/gh-pages/pyguide.md).
+
+# General Guidelines
+
+- If you’re just getting started work on an issue labeled “First Timers Only” in any project. You can refer additional resources available on our [website](http://systers.io/).
+- In an active repository (not an archived one), choose an open issue from the issue list, claim it in the comments or ask around in [zulip](https://anitab-org.zulipchat.com/), and a maintainer will assign it to you.
```

Comment:
I guess the zulip part here is not necessary here. What say @keshakaneria ?
But if you want to mention it you can write **or ask in the specified project channel on zulip**

Reply:
Yes, since zulip link is already mentioned once above, let us not mention it again here. You can replace the sentence as `claim it in the comments section or discuss it in their respective #questions stream on zulip and the maintainer will assign the issue to you.`